CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ pay attention to a few things:
 
 # Coding and documentation Style
 
-- Code must be formated with `yarn run format`
+- Code must be formatted with `yarn run format`
```

Comment:
**nitpick:** Changes should be extracted into its on commit `docs: remove typo in CONTRIBUTING.md file` (as an example)